### PR TITLE
Add SQL statement support

### PR DESCRIPTION
**Pull request by mosabua**
As discussed with @electrum @Ordinant and @Jessie212  .. to improve the SQL support sections and provide more general info and guidance. 

I wasn't sure where to place the file and how to best hook it into the overall flow and info .. this is a base for discussion and to get started. Please provide some first feedback so we can move this forward and ideally get merged soon and implement in many other connectors.

Also fyi @findepi

### CODE REVIEW

File: docs/src/main/sphinx/language/overview.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+* :doc:`/sql/grant-roles`
+* :doc:`/sql/grant`
+* :doc:`/sql/revoke-roles`
+* :doc:`/sql/revoke`
+* :doc:`/sql/set-role`
+* :doc:`/sql/show-grants`
+* :doc:`/sql/show-role-grants`
+* :doc:`/sql/show-roles`
+
+Transactions
```
Comment (mosabua) [inline docs/src/main/sphinx/language/overview.rst:109]: I wasnt sure .. but if this is correct then we will need to add a transactions section for connectors that support it.

File: docs/src/main/sphinx/language/overview.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
+=====================
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into a number of topics.
+Numerous statements are part of the core engine and therefore available in all
+use cases. On the other hand the details and architecture of the connected data
+sources prevent some SQL functionality.
```
Comment (hashhar) [inline docs/src/main/sphinx/language/overview.rst:8]: Maybe
```suggestion
connectors. On the other hand the details and architecture of the connected data
sources may prevent some SQL functionality.
```

File: docs/src/main/sphinx/language/overview.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+For example, if the data source does not support any write operations, Trino can
+not implement support for SQL statements like :doc:`/sql/delete`. Similarly if
+the underlying system does have any security concepts SQL statements like
+:doc:`/sql/create-role` can not provided by Trino and the connector.
```
Comment (hashhar) [inline docs/src/main/sphinx/language/overview.rst:13]: This is technically true - however can mislead people into thinking if their MySQL has users and roles support then Trino can manage them too.

File: docs/src/main/sphinx/language/overview.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+* :doc:`/sql/execute`
+* :doc:`/sql/explain`
+* :doc:`/sql/explain-analyze`
+* :doc:`/sql/prepare`
+* :doc:`/sql/reset-session`
+* :doc:`/sql/set-session`
+* :doc:`/sql/set-time-zone`
+* :doc:`/sql/show-functions`
+* :doc:`/sql/show-session`
+* :doc:`/sql/use`
```
Comment (hashhar) [inline docs/src/main/sphinx/language/overview.rst:36]: @electrum Should `VALUES` and `MATCH_RECOGNIZE` be part of this too? After all they are engine concepts and connectors don't participate in them.

**Review by hashhar — COMMENTED** [review]
Looks good. Some comments. This also partially addresses my concerns about how we were planning to know what is the overall supported set of SQL statements in Trino.

It might also be nice to nice to mention something to the effect of `Please consult the connector specific documentation` where we say vague things like `they are available with most connectors`.

File: docs/src/main/sphinx/connector/postgresql.rst [hunk 5]
```diff
@@ -125,14 +125,16 @@ SQL support
...
-and provides further details.
+metadata in PostgreSQL. The following section lists supported SQL statements and
+provides further details.
 
 .. important::
 
-    No other SQL statements are supported.
+    Only SQL statements in the following lists and the :ref:`globally available
+    statements <sql-globally-available>` are supported. No other SQL statements
+    are supported.
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/postgresql.rst:135]: This connector supports the following SQL statements in addition to the :ref:`globally available statements <sql-globally-available>`. No other SQL statements are supported.

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 6]
```diff
@@ -120,13 +120,16 @@ SQL support
...
+metadata in SQL Server. The following section lists supported SQL statements and
+provides further details.
 
 .. important::
-  No other SQL statements are supported.
 
-Read access:
+    Only SQL statements in the following lists and the :ref:`globally available
+    statements <sql-globally-available>` are supported. No other SQL statements
+    are supported.
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/sqlserver.rst:130]: This connector supports the following SQL statements in addition to the :ref:globally available statements <sql-globally-available>. No other SQL statements are supported.

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 7]
```diff
@@ -84,14 +84,16 @@ SQL support
...
+metadata in Iceberg. The following section lists supported SQL statements and
+provides further details.
 
-  .. important::
+.. important::
 
-      No other SQL statements are supported.
+    Only SQL statements in the following lists and the :ref:`globally available
+    statements <sql-globally-available>` are supported. No other SQL statements
+    are supported.
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/iceberg.rst:94]: This connector supports the following SQL statements in addition to the :ref:globally available statements <sql-globally-available>. No other SQL statements are supported.

**Review by jhlodin — COMMENTED** [review]
Proposed alternate wording to the supported SQL statements blurb.

File: docs/src/main/sphinx/language/overview.rst (hunk 2 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:8]: Done

File: docs/src/main/sphinx/language/overview.rst (hunk 3 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:13]: Reworded and clarified

File: docs/src/main/sphinx/language/overview.rst (hunk 4 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:36]: They are part of read operations .. since they are both part of SELECT statements and are listed in that section therefore. E.a .. no select support also means no match recognize support.

File: docs/src/main/sphinx/language/overview.rst [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+<sql-read-operations>`, :ref:`write operations <sql-write-operations>`,
+:ref:`security operations <sql-security-operations>` and :ref:`transactions
+<sql-security-transactions>`.
+
+.. _sql-globally-available:
+
+Globally available statements
+-----------------------------
+
+The following statements are implemented in the core engine and available with
```
Comment (mosabua) [inline docs/src/main/sphinx/language/overview.rst:30]: We could also break this up a bit into the following 

* session stuff
* prepared statement stuff
* query/explain stuff 

but I am not sure its worth doing that

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 9]
```diff
@@ -84,14 +84,17 @@ SQL support
...
+.. important::
 
-      No other SQL statements are supported.
+    The connector supports the following SQL statements in addition to the
+    :ref:`globally available statements <sql-globally-available>`. No other SQL
+    statements are supported.
 
-Read access:
+
+Read operations:
```
Comment (electrum) [inline docs/src/main/sphinx/connector/iceberg.rst:97]: Remove, these are always available

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 10]
```diff
@@ -84,14 +84,17 @@ SQL support
 -----------
 
 The connector provides **read access and limited write access** to data and
-metadata in Iceberg. The following section lists all supported SQL statements
-and provides further details.
+metadata in Iceberg. The following section lists supported SQL statements and
+provides further details.
 
-  .. important::
+.. important::
```
Comment (electrum) [inline docs/src/main/sphinx/connector/iceberg.rst:90]: I don't think we need to use "important" here, as it's just information like any other, and having this formatting in every connector will appear cluttered. We don't say "important: here are the requirements", rather just "requirements".

File: docs/src/main/sphinx/language/overview.rst [hunk 11]
```diff
@@ -0,0 +1,133 @@
+=====================
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into a number of topics.
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:5]: a number of -> several

File: docs/src/main/sphinx/language/overview.rst [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+Numerous statements are part of the core engine and therefore available in all
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:6]: Numerous -> Many

File: docs/src/main/sphinx/language/overview.rst [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+use cases. For example, you can always set session properties or inspect an
+explain plan and perform other actions with the :ref:`globally available
+statements <sql-globally-available>`.
+
+On the other hand the details and architecture of the connected data sources may
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:11]: "On the other hand" seems to be a colloquialism that might not make sense to non-native readers

File: docs/src/main/sphinx/language/overview.rst [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+prevent some SQL functionality. For example, if the data source does not support
+any write operations, Trino can not implement support for SQL statements like
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:13]: "Trino cannot implement" sounds strange. Maybe more direct

> For example, if the data source does not support write operations, then the DELETE statement cannot be executed against the data source.

File: docs/src/main/sphinx/language/overview.rst [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+:doc:`/sql/delete`.
+
+Similarly if the underlying system does not have any security concepts, SQL
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:16]: Comma after "similarly"

File: docs/src/main/sphinx/language/overview.rst [hunk 16]
```diff
@@ -0,0 +1,133 @@
...
+statements like :doc:`/sql/create-role` can not provided by Trino and the
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:17]: There should be a better way to phrase this

File: docs/src/main/sphinx/language/overview.rst [hunk 17]
```diff
@@ -0,0 +1,133 @@
...
+* :doc:`/sql/show-grants`
+* :doc:`/sql/show-role-grants`
+* :doc:`/sql/show-roles`
+
+.. _sql-security-transactions:
+
+Transactions
+------------
+
+The following statements manage transactions. Availability varies widely from
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:126]: How about

> Most connectors do not support transactions.

File: docs/src/main/sphinx/language/overview.rst [hunk 18]
```diff
@@ -0,0 +1,133 @@
...
+connector to connector, and details are available with the documentation for
+each connector:
+
+* :doc:`/sql/call`
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:130]: This is not related to transactions. It is used to execute connector specific procedures, typically for write or metadata operations that cannot be expressed as SQL. See Hive connector for examples.

File: docs/src/main/sphinx/language/overview.rst [hunk 19]
```diff
@@ -0,0 +1,133 @@
...
+* :doc:`/sql/update`
+
+.. _sql-security-operations:
+
+Security operations
+-------------------
+
+The following statements provide security-related operations to security
+configuration, data and meta data exposed by by a connector plugin accessing a
+data source. Availability varies widely from connector to connector, and details
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:107]: Something like
 
> Most connectors do not support these operations. Connectors that support them will document it explicitly.

(phrasing could use improvement)

File: docs/src/main/sphinx/language/overview.rst [hunk 20]
```diff
@@ -0,0 +1,133 @@
...
+by a connector plugin accessing a data source. Availability varies widely from
+connector to connector, and details are available with the documentation for
+each connector:
+
+* :doc:`/sql/alter-schema`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/alter-view`
+* :doc:`/sql/comment`
+* :doc:`/sql/create-schema`
+* CREATE MATERIALIZED VIEW
```
Comment (electrum) [inline docs/src/main/sphinx/language/overview.rst:83]: Code quotes

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 21]
```diff
@@ -120,13 +120,16 @@ SQL support
...
 
 .. important::
-  No other SQL statements are supported.
 
-Read access:
+    The connector supports the following SQL statements in addition to the
+    :ref:`globally available statements <sql-globally-available>`. No other SQL
+    statements are supported.
+
+Read operations:
```
Comment (electrum) [inline docs/src/main/sphinx/connector/sqlserver.rst:132]: Remove

File: docs/src/main/sphinx/connector/postgresql.rst [hunk 22]
```diff
@@ -125,14 +125,16 @@ SQL support
...
 
 .. important::
 
-    No other SQL statements are supported.
+    The connector supports the following SQL statements in addition to the
+    :ref:`globally available statements <sql-globally-available>`. No other SQL
+    statements are supported.
 
-Read access:
+Read operations:
```
Comment (electrum) [inline docs/src/main/sphinx/connector/postgresql.rst:137]: Remove

File: docs/src/main/sphinx/language/overview.rst (hunk 4 above)
Comment (findepi) [reply docs/src/main/sphinx/language/overview.rst:46]: Then we would want to document that connectors support order by, window functions, group by, etc.

Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:46]: well... they are part of the same doc page for select and as such as linked. The values and match_recognize files show up as separate entities in the SQL syntax. If we drop linking to it we should follow up with doc updates on select, values and match_recognize to explain that they all work from the same select ..

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 9 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/iceberg.rst:97]: all of them? For all connectors? And then also not mention them in the connectors each but instead say that all read operations as also supported?

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 10 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/iceberg.rst:90]: In all our feedback on the changes along this that are already live we heard that the fact that no other statements are supported is very important. We originally used note and then upgrade to warning. That was too much though and we found important to work well.

File: docs/src/main/sphinx/language/overview.rst (hunk 11 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:5]: done

File: docs/src/main/sphinx/language/overview.rst (hunk 13 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:11]: Replaced with "However" ... but I am not too happy with that either. Any other ideas?

File: docs/src/main/sphinx/language/overview.rst (hunk 15 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:16]: added

File: docs/src/main/sphinx/language/overview.rst (hunk 16 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:17]: Reworded

File: docs/src/main/sphinx/language/overview.rst (hunk 17 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:126]: done

File: docs/src/main/sphinx/language/overview.rst (hunk 19 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:107]: I removed the second part and added it to the overall section in the beginning instead so we don't repeat it everywhere.

File: docs/src/main/sphinx/language/overview.rst (hunk 20 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:83]: Done here and with the others

File: docs/src/main/sphinx/connector/sqlserver.rst (hunk 21 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/sqlserver.rst:132]: Same question.. does that mean they all work for all connectors.. then we should reword the important box..

File: docs/src/main/sphinx/language/overview.rst (hunk 18 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst]: oops .. its already in the global ones anyway. Removed

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 10 above)
Comment (electrum) [reply docs/src/main/sphinx/connector/iceberg.rst:90]: None of these changes are live yet (in a released version). I'm not convinced that this is more important than other things in the connector documentation, and having this same "important" note for every connector decreases the value of such notifications.

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 9 above)
Comment (electrum) [reply docs/src/main/sphinx/connector/iceberg.rst:97]: We can have a one sentence description (max) in the boilerplate section for the connector and discuss it more on the linked overview page. We don't need to list every read operation on the page for every connector.

File: docs/src/main/sphinx/connector/sqlserver.rst (hunk 21 above)
Comment (electrum) [reply docs/src/main/sphinx/connector/sqlserver.rst:132]: They work for all connectors

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 9 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/iceberg.rst:97]: I think I got a nice solution now

File: docs/src/main/sphinx/connector/sqlserver.rst (hunk 21 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/sqlserver.rst:132]: okay .. changed accordingly

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 10 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/iceberg.rst]: Discussed in call and updated as decided with @dain and @electrum

File: docs/src/main/sphinx/language/overview.rst [hunk 23]
```diff
@@ -0,0 +1,159 @@
...
+.. _sql-write-views:
+
+Views
+^^^^^
+
+* :doc:`/sql/create-view`
+* :doc:`/sql/drop-view`
+* :doc:`/sql/alter-view`
+
+
```
Comment (jhlodin) [inline docs/src/main/sphinx/language/overview.rst:124]: Extra line

File: docs/src/main/sphinx/language/overview.rst [hunk 24]
```diff
@@ -0,0 +1,159 @@
...
+* :doc:`/sql/revoke`
+
+.. _sql-security-transactions:
+
+Transactions
+------------
+
+The following statements manage transactions. Most connectors do not support
+transactions. Details are available with the documentation for each
+connector:
```
Comment (jhlodin) [inline docs/src/main/sphinx/language/overview.rst:155]: "Details are available with the documentation for each connector:"

If we're including this snippet here, we should include it with all the other section introductions that say that support is limited to some connectors. If we can't do that, because not all the connector docs have a list of supported SQL statements yet, then I don't think we should have this line at all until that's the case.

File: docs/src/main/sphinx/language/overview.rst [hunk 25]
```diff
@@ -0,0 +1,159 @@
...
+* :doc:`/sql/alter-view`
+
+
+.. _sql-security-operations:
+
+Security operations
+-------------------
+
+The following statements provide security-related operations to security
+configuration, data and meta data exposed by a connector accessing a data
```
Comment (jhlodin) [inline docs/src/main/sphinx/language/overview.rst:131]: data, and meta

File: docs/src/main/sphinx/language/overview.rst (hunk 24 above)
Comment (jhlodin) [reply docs/src/main/sphinx/language/overview.rst:155]: You also have a nearly identical sentence above all of these sections at line 25, so you could just remove this from here as redundant.

File: docs/src/main/sphinx/language/overview.rst [hunk 26]
```diff
@@ -0,0 +1,159 @@
...
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into several topics. Many
+statements are part of the core engine and therefore available in all use cases.
+For example, you can always set session properties or inspect an explain plan
+and perform other actions with the :ref:`globally available statements
+<sql-globally-available>`.
+
+However the details and architecture of the connected data sources may prevent
```
Comment (jhlodin) [inline docs/src/main/sphinx/language/overview.rst:11]: However,

Comment (jhlodin) [inline docs/src/main/sphinx/language/overview.rst:11]: Would prefer "can limit" to "may prevent", the latter sounds like the data source is somehow taking action against the SQL functionality.

File: docs/src/main/sphinx/language/overview.rst [hunk 27]
```diff
@@ -0,0 +1,159 @@
...
+some SQL functionality. For example, if the data source does not support any
+write operations, then a :doc:`/sql/delete` cannot be executed against the data
```
Comment (jhlodin) [inline docs/src/main/sphinx/language/overview.rst:13]: /sql/delete` statement cannot be executed

**Review by jhlodin — APPROVED** [review]

File: docs/src/main/sphinx/language/overview.rst (hunk 23 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:124]: removed

File: docs/src/main/sphinx/language/overview.rst (hunk 24 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:155]: good point... made it all consistent now

File: docs/src/main/sphinx/language/overview.rst (hunk 26 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:11]: done

File: docs/src/main/sphinx/language/overview.rst (hunk 25 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:131]: done

File: docs/src/main/sphinx/language/overview.rst (hunk 26 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:11]: done

File: docs/src/main/sphinx/language/overview.rst (hunk 27 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/overview.rst:13]: done

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 28]
```diff
@@ -83,42 +83,15 @@ At a minimum, ``hive.metastore.uri`` must be configured:
...
-* :doc:`/sql/describe`
-
-Write access:
-
-* :doc:`/sql/alter-table`
-* :doc:`/sql/create-schema`
-* CREATE MATERIALIZED VIEW
-* :doc:`/sql/create-table`, see also :ref:`iceberg-tables`
-* :doc:`/sql/create-table-as`, see also :ref:`iceberg-tables`
+The connector provides read access and write access to data and metadata in
```
Comment (dain) [inline docs/src/main/sphinx/connector/iceberg.rst:86]: `This connector`?

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 29]
```diff
@@ -83,42 +83,15 @@ At a minimum, ``hive.metastore.uri`` must be configured:
...
+Iceberg. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
+
+* :ref:`sql-write-schema-table`, see also :ref:`iceberg-tables`
```
Comment (dain) [inline docs/src/main/sphinx/connector/iceberg.rst:91]: I'd always put `insert`, `update`, and `delete` first.  Then maybe views, and finally `schema and table management` and security bits

File: docs/src/main/sphinx/language/sql-support.rst [hunk 30]
```diff
@@ -0,0 +1,157 @@
...
+.. _sql-write-data:
+
+Data management
+^^^^^^^^^^^^^^^
+
+* :doc:`/sql/insert`
+* :doc:`/sql/update`
+* :doc:`/sql/delete`
+
+.. _sql-write-materialized-views:
```
Comment (dain) [inline docs/src/main/sphinx/language/sql-support.rst:92]: I don't think the anchor needs `write-``

File: docs/src/main/sphinx/language/sql-support.rst [hunk 31]
```diff
@@ -0,0 +1,157 @@
...
+
+Materialized views
+^^^^^^^^^^^^^^^^^^
+
+* ``CREATE MATERIALIZED VIEW``
+* ``DROP MATERIALIZED VIEW``
+* ``REFRESH MATERIALIZED VIEW``
+
+.. _sql-write-schema-table:
```
Comment (dain) [inline docs/src/main/sphinx/language/sql-support.rst:101]: I'd name the anchor `sql-schema-table-management` or such

File: docs/src/main/sphinx/language/sql-support.rst [hunk 32]
```diff
@@ -0,0 +1,157 @@
...
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/alter-schema`
+* :doc:`/sql/comment`
+
+.. _sql-write-views:
```
Comment (dain) [inline docs/src/main/sphinx/language/sql-support.rst:115]: maybe just `sql-views`.. or maybe `-management` to match schema and table above

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 29 above)
Comment (electrum) [reply docs/src/main/sphinx/connector/iceberg.rst:91]: Sounds good to me

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 28 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/iceberg.rst:86]: done

File: docs/src/main/sphinx/connector/iceberg.rst (hunk 29 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/iceberg.rst:91]: done

File: docs/src/main/sphinx/language/sql-support.rst (hunk 31 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/sql-support.rst:101]: done

File: docs/src/main/sphinx/language/sql-support.rst (hunk 32 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/sql-support.rst:115]: made consistent with all others

File: docs/src/main/sphinx/language/sql-support.rst (hunk 30 above)
Comment (mosabua) [reply docs/src/main/sphinx/language/sql-support.rst:92]: done

**Review by electrum — APPROVED** [review]

**Comment by mosabua** [comment]
Thanks for the help everyone and merge @electrum !
